WordPress/Classes/ViewRelated/NUX/NUXAbstractViewController.swift

```diff
@@ -0,0 +1,249 @@
+import UIKit
+import WordPressComAnalytics
+import WordPressShared
+
+/// A base class for the various NUX related related view controllers. 
+/// The base class sets up and configures common functionality, such as the help 
+/// button and badge.
+/// It is assumed that NUX controllers will always be presented modally.
+///
+class NUXAbstractViewController : UIViewController
```

Comment:
The original approach used parent/child view controllers.  It was a bit problematic in that we were maintaining more state than desired, wrangle a back button, and needing to manually manage the navigation history. We switching to a UINavigationController to simplify things and let it do the job it was intended for.